### PR TITLE
Update footer for 2019

### PR DESCRIPTION
**Pull request by DanielNi**

### CODE REVIEW

File: index.mustache [hunk 1]
```diff
@@ -115,14 +115,12 @@
       <nav class="footer-navigation">
         <i class="icon-square-footer"></i>
         <ul class="footer-navigation-list">
-          <li class="footer-navigation-item"><a href="https://squareup.com/directory">Square Directory</a></li>
-          <li class="footer-navigation-item"><a href="https://cash.me">Square Cash</a></li>
-          <li class="footer-navigation-item"><a href="https://squareup.com/register">Square Register</a></li>
```
Comment (zach-klippenstein) [inline index.mustache]: Did you mean to remove all these links too?

Comment (mootrichard) [reply index.mustache]: Cash has their own GitHub, and `/register` redirects to `/point-of-sale`. 

I guess we could add in a link to the Cash GitHub in there if we really wanted.

Comment (DanielNi) [reply index.mustache]: Added a link to the Cash Github.

Comment (zach-klippenstein) [reply index.mustache]: Cash has their own github, but no reason to not point people there from this page.

**Pull request merged**